packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-api/index.md

```diff
@@ -110,6 +110,13 @@ A best practice is to use a generated globally unique identifier (GUID) for this
 
 Okta user management requires that your SCIM API supports an `active` attribute for each user resource that can be set to `true` or `false` to denote a resource as "active" or "inactive".
 
+## Example implementations
+
+There are numerous examples of how to implement SCIM servers and applications. Here are a few to get you started:
+
+- [scimify](https://toolkit.okta.com/apps/scimify/) - A PHP application that supports both SCIM 1.1 and SCIM 2.0 servers with operations for /Users, /Groups and /ServiceProviderConfig endpoints
```

Comment:
Apache SCIMple is good one but we shouldn't link scimify as reference implementation. It's code is not production quality.

Reply:
Other alternative,
https://code.google.com/archive/p/openscim/

Reply:
Examples updated based on conversation with Mrunal in slack.